### ISSUE TITLE
Some message may be lost (not synced) when switching between browsers

### ISSUE THREAD

**Issue by rachelmcr** [tinkertinker/wp-calypso#41]
#### Steps to reproduce
1. In a desktop browser, starting at URL: https://wordpress.com/
2. Click "Support Chat" to start a new chat
3. Enter a message and start chatting with the operator
4. In a mobile browser, open https://wordpress.com/me/chat
5. Continue chatting with the operator (note the previous messages from the desktop browser won't appear in the mobile browser, per #16, but new messages should appear in both browsers)
6. Let both chat windows sit for a minute or so
7. Start typing in the desktop browser again
#### What I expected

I expected all messages to be synced across browsers once I was chatting in both, so any new messages typed on desktop would appear on mobile, and vice versa.
#### What happened instead

The first message I typed after pausing and then resuming the chat on desktop didn't appear in my mobile browser. All messages after that were synced as expected, even when I switched quickly between browsers.

(I'm not sure if the pause (step 6 above) was the cause of the unsynced message, but it's the only other factor I could identify. I have not reproduced the issue since the first time it happened.)
#### Browser / OS version

Desktop: Chrome 52.0.2743.116 / Mac OS X 10.11.6
Mobile: iPhone 6, iOS 9.3.5, Chrome app
#### Screenshot / Video

Chat in the desktop browser:

![screen shot 2016-08-31 at 6 43 53 pm](https://cloud.githubusercontent.com/assets/8658164/18140159/28dc7e64-6fac-11e6-9db1-c4bf88e717cf.png)

Chat in the mobile browser:

![2016-08-31 18 44 15](https://cloud.githubusercontent.com/assets/8658164/18140208/60a1e67c-6fac-11e6-8783-138b3025fc06.png)
#### Context / Source
# manual-testing

**Comment by andreabadgley** [comment]
@rachelmcr This looks similar to https://github.com/tinkertinker/wp-calypso/issues/16 , but I may be missing something. Would you say this report is a different issue from #16?

**Comment by rachelmcr** [comment]
Here's a key observation from #16:

> Current (existing) messages will not sync into a new session in a second browser. However, once the new session is opened, all subsequent messages will sync across browsers.

This issue is different, in that I already had the session going on both browsers but not all of the subsequent messages synced correctly. Quick recap:
1. I started a chat on desktop
2. I continued the chat on mobile (with the desktop browser still open)
3. I returned to desktop to continue the chat (with the mobile browser still open)
4. My first message from desktop in step 3 didn't appear on mobile

That said, if the underlying cause is the same for both issues then this may be a dupe of #16.
